docs/plugins/clean-url-tracker.md

```diff
@@ -0,0 +1,146 @@
+    <th align="left">Type</th>
+    <th align="left">Default</th>
+  </tr>
+  <tr valign="top">
+    <td><code>stripQuery</code></a></td>
+    <td><code>boolean</code></a></td>
+    <td>
+      When <code>true</code>, the query string portion of the URL will be removed.<br>
+      <strong>Default:</strong> <code>false</code>
+    </td>
+  </tr>
+  <tr valign="top">
+    <td><code>queryDimensionIndex</code></a></td>
+    <td><code>number</code></a></td>
+    <td>
+      There are cases where you want to strip the query string from the URL, but you still want to record what query string was originally there, so you can report on those values separately. You can do this by creating a new <a href=""https://support.google.com/analytics/answer/2709829>custom dimension</a> in Google Analytics. Set the dimension's <a href="https://support.google.com/analytics/answer/2709828#example-hit">scope</a> to "hit", and then set the index of the newly created dimension as the <code>queryDimensionIndex</code> option. Once set, the stripped query string will be set on the custom dimension at the specified index.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td><code>indexFilename</code></a></td>
+    <td><code>string</code></a></td>
+    <td>
+      When set, the <code>indexFilename</code> value will be stripped from the end of a URL. If your server supports automatically serving index files, you should set this to whatever value your server uses (usually <code>'index.html'</code>).
+    </td>
+  </tr>
+  <tr valign="top">
+    <td><code>trailingSlash</code></a></td>
+    <td><code>string</code></a></td>
+    <td>
+      When set to <code>'add'</code>, a trailing slash is appended to the end of all URLs (if not already present). When set to <code>'remove'</code>, a trailing slash is removed from the end of all URLs. No action is taken if any other value is used. Note: when using the <code>indexFilename</code> option, index filenames are stripped prior to the trailing slash being added or removed.
+    </td>
+  </tr>
+</table>
+
+## Methods
+
+The following table lists all methods for the `cleanUrlTracker` plugin:
+
+<table>
+  <tr valign="top">
+    <th align="left">Name</th>
+    <th align="left">Description</th>
+  </tr>
+  <tr valign="top">
+    <td><code>remove</code></a></td>
+    <td>Removes the <code>cleanUrlTracker</code> plugin from the specified tracker and restores all modified tasks to their original state prior to the plugin being required.</td>
+  </tr>
+</table>
+
+For details on how `analytics.js` plugin methods work and how to invoke them, see [calling plugin methods](https://developers.google.com/analytics/devguides/collection/analyticsjs/using-plugins#calling_plugin_methods) in the `analytics.js` documentation.
+
+## Example
+
+Given the four URL paths shown in the table at the beginning of this guide, the following `cleanUrlTracker` configuration would ensure that only the URL path `/contact` ever appears in your reports.
```

Comment:
maybe be explicit about queryDimensionIndex value and say for custom dimension witg index 1.